lib/OpenQA/Schema/Result/Users.pm

```diff
@@ -51,6 +51,14 @@ __PACKAGE__->add_columns(
         false_id      => ['0', '-1'],
         default_value => '0',
     },
+    last_login_version => {
+        data_type   => 'text',
```

Comment:
I wouldn't make this a text version - make a list of features to be presented and store the number seen. And then have the user either ignore them all or go through them.